conceptnet5/vectors/evaluation/bias.py

```diff
@@ -196,15 +195,16 @@ def measure_bias(frame):
     - Coarse-grained ethnicity
     - Religious beliefs
     """
-    vsw = VectorSpaceWrapper(frame=frame)
-    vsw.load()
+    # Assume English for DataFrames that don't have language tags.
+    if '/c/' not in frame.index[0]:
+        frame.index = ['/c/en/%s' % label for label in frame.index]
```

Comment:
Good point - I made both of those changes and it still works.

(I am sad about `frame.index[1].startswith('/c/')`, and I wish I knew where the label `/` was coming from so that it wasn't necessary.)